includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -109,6 +109,22 @@ public static function register() {
 		}
 	}
 
+	/**
+	 * Determine whether the admin menu item should be included.
+	 *
+	 * @return bool Whether to show in menu.
+	 */
+	public static function should_show_in_menu() {
+		global $pagenow;
+		if ( current_theme_supports( 'amp' ) ) {
+			return true;
+		}
+		if ( 'edit.php' === $pagenow && ( isset( $_GET['post_type'] ) && self::POST_TYPE_SLUG === $_GET['post_type'] ) ) { // WPCS: CSRF OK.
```

Comment:
Why not return the result of the conditional instead of explicitedly retuning `true` or `false`?

Reply:
Sure. Done in a83a569f.